src/io/filesystem/CMakeLists.txt

```
@@ -17,7 +17,7 @@ wl_library(io_filesystem
     base_exceptions
     base_macros
     io_stream
-    third_party_minizip
+    ${MINIZIP_STATIC_LIBRARIES}
```

```suggestion
    MINIZIP_STATIC_LIBRARIES
```
Should resolve the codecheck problem. Like the other library names it does not need varible expansion.
However you should switch the includes in zip_filesystem.h to the system headers (`<zip.h>`, `<unzip.h>`) somehow when the external minizip is available, otherwise you will still get a codecheck error.

---

> Should resolve the codecheck problem. Like the other library names it does not need varible expansion.

The build fails when I remove the variable expansion:

```
[31/189] Linking CXX executable src/base/test/test_base
FAILED: src/base/test/test_base 
...
usr/bin/ld: cannot find -lMINIZIP_STATIC_LIBRARIES: No such file or directory
```